### PR TITLE
Added appveyor yml file

### PR DESCRIPTION
**Pull request by wvangeit**

### CODE REVIEW

File: efel/cppcore/LibV3.cpp [hunk 1]
```diff
@@ -1004,7 +1005,8 @@ static int __AP_fall_indices(const vector<double>& v, const vector<int>& apbi,
     vector<double> vpeak(&v[pi[i]], &v[apei[i]]);
     transform(vpeak.begin(), vpeak.end(), vpeak.begin(),
               bind2nd(std::minus<double>(), halfheight));
-    transform(vpeak.begin(), vpeak.end(), vpeak.begin(), std::ptr_fun(::fabs));
+    transform(vpeak.begin(), vpeak.end(), vpeak.begin(), 
+              static_cast<double(*)(double)>(fabs));
```
Comment (mgeplf) [inline efel/cppcore/LibV3.cpp]: this is weird, what's the vs compiler error?

File: efel/cppcore/LibV5.cpp [hunk 2]
```diff
@@ -1810,8 +1810,8 @@ static int __AP_phaseslope(const vector<double>& v, const vector<double>& t,
     apbegin_index = apbi[i];
     range_min_index = apbegin_index - int(range);
     range_max_index = apbegin_index + int(range);
-    if (range_min_index < 0 or range_max_index < 0) return -1;
-    if (range_min_index > (int)t.size() or range_max_index > (int)t.size())
+    if (range_min_index < 0 || range_max_index < 0) return -1;
```
Comment (mgeplf) [inline efel/cppcore/LibV5.cpp]: wow, that's been around for a while
surprised xlc can handle 'or'

File: efel/cppcore/LibV3.cpp (hunk 1 above)
Comment (wvangeit) [inline efel/cppcore/LibV3.cpp]: This kind of error: https://ci.appveyor.com/project/wvangeit/efel/build/1.0.16

File: efel/cppcore/LibV5.cpp (hunk 2 above)
Comment (wvangeit) [inline efel/cppcore/LibV5.cpp]: Was probably writing Python code right before adding this line :-)

File: efel/cppcore/LibV3.cpp (hunk 1 above)
Comment (wvangeit) [inline efel/cppcore/LibV3.cpp]: What's apparently happening is that ::fabs is overloaded a couple of times for floats, doubles, etc. and that the compiler couldn't decide which one to pick

Comment (mgeplf) [inline efel/cppcore/LibV3.cpp]: instead of static cast, can't we just template 
std::ptr_fun<double, double>(::fabs)

Comment (wvangeit) [inline efel/cppcore/LibV3.cpp]: doesn't the static cast work ?

Comment (mgeplf) [inline efel/cppcore/LibV3.cpp]: I generally try and avoid casts, personally

**Comment by wvangeit** [comment]
Closing this for now, until we have a need for this

**Pull request closed without merging**